src/debug/ee/arm64/arm64walker.cpp

```diff
@@ -178,17 +177,72 @@ BYTE*  NativeWalker::SetupOrSimulateInstructionForPatchSkip(T_CONTEXT * context,
     else if(RegNum >= 0)
     {
         CORDbgSetInstruction((CORDB_ADDRESS_TYPE *)patchBypass, 0xd503201f); //Add Nop in buffer
+
         PCODE RegContents;
-        if ((opcode & 0xFF000000) == 0x58000000) //Actual LDR literal
+        if ((opcode & 0x3BF000000) == 0x18000000) //LDR Literal
         {
-            RegContents = GetMem(ip);
+            if ((opcode & 0x4000000)) //LDR literal for SIMD
+            {
+                LOG((LF_CORDB, LL_INFO100000, "Arm64Walker::Simulate opcode: %x to LDR V%d %p\n", opcode, RegNum, offset));
+                short opc = (opcode >> 30);
+                switch (opc)
+                {
+                case 0: //4byte data into St
+                case 1: //8byte data into Dt
+                    RegContents = (PCODE)GetMem(ip);
+                    SetReg(context, RegNum, RegContents);
+                    break;
+                case 2: //SIMD 16 byte data
+                    NEON128 SimdRegContents = GetSimdMem(ip);
+                    SetSimdReg(context, RegNum, SimdRegContents);
+                    break;
+                default:
+                    LOG((LF_CORDB, LL_INFO100000, "Arm64Walker::Simulate Unknown opcode: %x [LDR(litera,SIMD &FP)]  \n", opcode));
+                    _ASSERTE(!("Arm64Walker::Simulated Unknown opcode"));
+
+                }
+            }
+            else 
+            {
+                short opc = (opcode >> 30);
+                switch (opc)
+                {
+                case 0: //4byte data into Wt
+                    LOG((LF_CORDB, LL_INFO100000, "Arm64Walker::Simulate opcode: %x to LDR W%d %p\n", opcode, RegNum, offset));
```

Comment:
The upper 32-bit need to be Zero-ed, so  this can't be the sane as the 8-byte case here